src/foam/blob/Blob.js

```diff
@@ -712,12 +712,10 @@ return file;`
 }
 
 this.setup(x);
-HashingOutputStream os = null;
+long size = blob.getSize();
+File tmp = allocateTmp(x, 1);
 
-try {
-  long size = blob.getSize();
-  File tmp = allocateTmp(x, 1);
-  os = new HashingOutputStream(new FileOutputStream(tmp));
+try(HashingOutputStream os = new HashingOutputStream(new FileOutputStream(tmp))) {
```

Comment:
Please follow the FOAM style guidelines:
https://github.com/foam-framework/foam/wiki/FOAM2-Style-Guidelines

Reply:
Fixed